tests/integration-tests/processors/processors-test.js

```diff
@@ -233,6 +260,47 @@ describe('Processors api', function() {
                 createProcessorResponse.statusCode.should.eql(422);
             });
         });
+        describe('POST /processors/{processor_id}/download', () => {
+            it('try for a raw_javascript processor', async function() {
+                const rawJSProcessor = generateRawJSProcessor('name');
+                const createProcessorResponse = await requestSender.createProcessor(rawJSProcessor, validHeaders);
+                should(createProcessorResponse.statusCode).equal(201);
+                const processorId = createProcessorResponse.body.id;
+                const response = await requestSender.updateDownloadProcessorContent(processorId);
+                should(response.statusCode).equal(400);
+            });
+
+            it('processor doesn\'t exist', async function() {
+                const processorId = uuid();
+
+                const deleteRes = await requestSender.deleteProcessor(processorId);
```

Comment:
no need to delete processor with newly generated uuid, altho it "promises" us that this processor wont exist for sure, we rely on the randomness of `uuid()`